src/modules/function.js

```diff
@@ -0,0 +1,21 @@
+// Getting started
+const inputField = document.querySelector('.input-field textarea');
+const todoLists = document.querySelector('.todoLists');
+// const pendingNum = document.querySelector('.pending-num');
+// const clearButton = document.querySelector('.clear-button');
```

Comment:
- [ ] Please do the same thing here. :+1:

Reply:
Hello, @MoyasiGinko I think it would be better if you remove them from here and put them in some text file on your local machine since you'll need them later. It would be great if this branch only contains pieces of code required for this milestone.

Reply:
Understood.  i think i can do that. thank you @amejid 
if you don't mind i would like to ask you for some guidance if you are free? i m stuck in an error and i don't know how to work it out.